### PR TITLE
NEUBURG-64: Add logging

### PR DESCRIPTION
**Pull request by NKlug**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
+.neuburg-backendrc
```
Comment (maxammann) [inline .gitignore]: This file should stay in the repository. Maybe put in in another directory like /templates/

File: src/services/ErrorService.js [hunk 2]
```diff
@@ -12,9 +13,11 @@ const errorTypes = {
   server: 'server'
 }
 
+const logger = log4js.getLogger()
```
Comment (maxammann) [inline src/services/ErrorService.js]: I tink this logger should be a member of the ErrorService class.

**Review by maxammann — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (NKlug) [reply .gitignore]: You mean that we put the "template" in the templates folder?  
Then `.neuburg-backenrc` would stay in the .gitignore right?

Comment (maxammann) [reply .gitignore]: Yes, I think that would make sense. Maybe also some docu about setting it up would make sense.

**Comment by maxammann** [comment]
Can we make the logging to file optional? Or if we don't need it disable it?

Logging to stdout/err is enough for our server setup.

**Comment by NKlug** [comment]
@maxammann I changed the setup in a way that when no log file is given in the rc, logging to a file is disabled.

**Review by maxammann — APPROVED** [review]

**Review by steffenkleinle — APPROVED** [review]

**Pull request merged**